labs/lab6/lab6.adoc

```diff
@@ -31,65 +30,9 @@ ip-10-10-0-60.ap-southeast-1.compute.internal   Ready     2h        v1.6.1+5115d
 
 == Expand the Cluster
 
-## Add Scaleup Job Templates
-
-Perform these steps from the Ansible Tower host
-
-## Create Job Template for ScaleUp Provision
-
-* Click "TEMPLATES" on the top navigation pane.
-** Click "+ADD", select "Job Template"
-** Provide a name of: ScaleUp-1-Provision
-** Click the "SEARCH" icon for the "INVENTORY" input box and select "OpenShift Inventory" and then click "SELECT".
-** Click the "SEARCH" icon for the "PROJECT" input box and select "Managing OCP from Install and Beyond" and then click "SELECT".
-** Click the "Choose a playbook" in the "PLAYBOOK" input box and select the "aws_add_node.yml" playbook.
-** Click the "SEARCH" icon for the "MACHINE CREDENTIAL" input box and select "RHTE SSH Machine" and then click "SELECT".
-** Click the "SEARCH" icon for the "SELECT CLOUD CREDENTIAL" input box and select "AWS" and then click "SELECT".
-** Add the following to the "EXTRA VARIABLES" pane.
-
-+
-[source, bash]
-----
-ec2_access_key: "{{ lookup('env', 'AWS_ACCESS_KEY') }}"
-ec2_secret_key: "{{ lookup('env', 'AWS_SECRET_KEY') }}"
-student_id: <student_id>
-openshift_cluster_public_url: "https{{':'}}//master-{{ student_id }}.{{ domain_name }}{{':'}}8443"
-----
+The _0-Self-Configure_ job template executed earlier configured a set of templates to automate the process to expand the OpenShift cluster.
```

Comment:
As previously discussed, you can always set these variables to `false` in `templates/tower_job_template_self_configure_extra_vars.yml.j2` if you want the student to configure this manually:

```
tower_job_template_scaleup_provision_config: true
tower_job_template_scaleup_install_config: true
tower_job_template_scaleup_configure_config: true
```

Reply:
A little too technical since we didnt go into detail about the underlying technical implementation